src/commands/create.ts

```diff
@@ -15,6 +15,7 @@ import {
     VariousUtils,
 } from '../misc';
 
+import { IServerInfo1, IServerInfo2 } from '../misc/interfaces';
```

Comment:
Please prefer changing these two interfaces (`IServerInfo1`, `IServerInfo2`) into pure `letters` format by their actual meanings instead of using `digit` suffix for naming. Even though sometimes it may be hard for us to find proper naming, but naming variables containing digits or using a single letter made your code hard to read.

Reply:
Done